rules/cross-platform/command_and_control_non_standard_ssh_port.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2022/10/18"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/10/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies adversaries (malicious process) who(which) may communicate using a port paring that are
+typically not associated with ssh. For example, ssh over port 2200 or port 2222 as opposed to the traditional port 22.
+Adversaries may make changes to the standard port used by a protocol to bypass filtering or
+muddle analysis/parsing of network data.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Non-Standard Port SSH connection(s)"
+references = [
+    "https://attack.mitre.org/techniques/T1571/"
+]
+risk_score = 47
+rule_id = "bc8ca7e0-92fd-4b7c-b11e-ee0266b8d9c9"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Command and Control", "macOS"]
+type = "eql"
+
+query = '''
+sequence by process.entity_id with maxspan=1m
+[process where event.action == "exec" and process.name:"ssh"]
```

Comment:
It would be nice if you could validate if we can build a detection using NPC that could determine if traffic is SSH without tying it to the ssh process directly. That could make it more resilient to bypasses that can simply change the process name